workbench/static/workbench/js/runtime/logger.js

```diff
@@ -37,7 +40,7 @@ XBlock generated them and other XBlock-local metadata.
             sendRequest, has;
 
         sendRequest = function(data, options) {
-	    console.log(data);
+	    console.log(JSON.stringify(data));
```

Comment:
Are you sure you need to stringify the object first? It should already be in JSON. Are you expecting nested objects inside the data object?